docs/developer/process/agent-release/pre-release.md

```diff
@@ -109,6 +109,12 @@ After all fixes have been cherry-picked:
 1. Push the changes to GitHub
 1. [Tag](#tag) with the appropriate `rc` number even if there were no changes
 
+After the RC build is done, manually run an [Agent Azure Pipeline](https://dev.azure.com/datadoghq/integrations-core/_build?definitionId=60) using the release branch, and the latest RC built. It will run all the e2e tests using a specific agent.
+
+!!! note
+    Image for Windows-Python 2 might not be built automatically for each RC. In order to build it, trigger the [Gitlab pipeline](https://github.com/DataDog/datadog-agent/blob/1b99fefa1d31eef8631e6343bdd2a4cf2b11f82d/.gitlab/image_deploy/docker_windows.yml#L43-L61).
```

Comment:
I'd be a bit more specific and add that you specifically need to trigger the `dev_branch-a6-windows` job that's in the `datadog-agent` pipeline that built the RC (whose link should have been shared by the release coordinator)

Reply:
Thanks for the details!